v2/component/runtime_info.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package component
+
+import (
+	"encoding/json"
+
+	pb "github.com/kubeflow/pipelines/v2/third_party/ml_metadata"
+)
+
+type inputParameter struct {
+	ParameterType  string
+	ParameterValue string
+}
+
+type inputArtifact struct {
+	// Where to read MLMD artifact. File is passed using Argo artifacts.
+	FileInputPath string
+
+	// The MLMD artifact.
+	Artifact *pb.Artifact `json:"-"`
+
+	// Generated by launcher.
+	// /tmp/launcher_component_inputs/<name>/data
+	LocalArtifactFilePath string `json:"-"`
```

Comment:
nit: shall we standardize a term for artifact file? What do you think about calling it payload?

so this becomes `PayloadPath` (I see no reason to repeat artifact,local here, because they are implied by context/the word path.)

Reply:
Ack. Same as above, thinking of doing this in separate PR. Left a TODO.